src/app/pages/news/11.3/02-inscription-view.md

```diff
@@ -1,6 +1,22 @@
 ## Inscription View {#inscriptionView}
 
-Massive design update, browser, finalization ...
+The new Inscription View is finally finished! 
+We've not only successfully migrated all the old SWT-based 
+masks and browsers into the new world but have also given the entire 
+interface an amazing redesign.
+
+- **UI update**:  Prepare to be amazed by the refreshed user interface! 
+Thanks to the sleek new design, navigating through inscriptions has never been 
+smoother, making oversight effortless. Among other things, the UI update features
+ collapsible sections for every subpart, improved tab naming and overall interface streamlining.
+- **Browsers**: The Browsers, a familier tool known from the LTS, have now 
```

Comment:
I have to read the full sentence until I get what you mean with a 'browser' ... and users will have their own assumption about what you refer to with a browser. So I'd start the other way around.

> We added assisting browsers to search for CMS entries, DataClass attributes, Java-Types and the like. 
... then some nice marketing sentence, to underline what the browser are for -> low code experience : e.g.
> The browsers make element configurations an easy and swift user-experience without the need to write any code.